reagurk/public/index.html

```diff
@@ -21,6 +21,23 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Gurkult</title>
+    <meta charSet="utf-8" />
+    <title>Gurkult | Open Source Community</title>
+    <meta name="description" content="The home for the Gurkans" />
```

Comment:
```suggestion
    <meta name="description" content="Learn open source by contributing to real world projects. Be a part of an inclusive, international community of developers." />
```
Sorry, I missed this one.